src/commands/deploy.ts

```diff
@@ -82,6 +82,22 @@ export async function deploy(this: IActionContext, target?: vscode.Uri | string
         throw new Error(localize('pythonNotAvailableOnWindows', 'Python projects are not supported on Windows Function apps.  Deploy to a Linux Consumption app.'));
     }
 
+    // we need this check due to this issue: https://github.com/Microsoft/vscode-azurefunctions/issues/625
+    if (node.isLinuxPreview) {
+        const applicationSettings: StringDictionary = await client.listApplicationSettings();
+        if (applicationSettings.properties && (applicationSettings.properties[WEBSITE_CONTENTAZUREFILECONNECTIONSTRING] || applicationSettings.properties[WEBSITE_CONTENTSHARE])) {
+            await ext.ui.showWarningMessage(
+                localize('noDeployWithWebsiteContent', 'App settings "{0}" and "{1}" are required for Portal editing, but prevent deployment.  Delete these settings and deploy?', WEBSITE_CONTENTAZUREFILECONNECTIONSTRING, WEBSITE_CONTENTSHARE),
```

Comment:
Screenshot of the message? I'm guessing it looks pretty ugly... 😬

Reply:
<img width="460" alt="screen shot 2018-10-11 at 12 59 20 pm" src="https://user-images.githubusercontent.com/5290572/46830489-99243000-cd55-11e8-82e6-bb0bb6ae828e.png">